### DISCUSSION TITLE
http causing a security issue

### DISCUSSION THREAD

**Discussion by rwjework** [e107inc/e107#5100] [Q&A Core] [answered]
I have SSL activated and was getting a not secure security message.  I ran a Dr Link check and got the following

URL
https://www.*********/e107/thumb.php?src=&aw=144&ah=144
Redirected to
https://placehold.it/144x144 HTTP 302
↪ https://www.placeholder.com/144x144 HTTP 301
↪ http://via.placeholder.com/144x144 HTTP 301

Does anyone know where this redirect to http might be in the code? This appears to be the only http link I have apart from the permanent redirect caused by enabling SSL.

**Comment by Alex-e107nl** [comment]
Goodmorning,
go to preferences - site information, if your Site Link Icon/Button and Site Logo are empty it gives that message... so i always make sure that there are 2 images in those fields :-)  I don't know where in the code the http comes from unfortunately... maybe this will help a bit?

Alex

**Reply by rwjework** [reply]
Thanks Alex. I have added a blank icon with letters on it to those two fields but I am still getting the error
Appreciate your help
Robert

**Reply by Jimmi08** [reply, marked as answer]
What e107 are you using? Latest one (from here, 2.3.3 I think) uses header('location: https://via.placeholder.com/'.$getsize);  in e_thumbnail_class.php 

empty src parameter means that your image is missing, so Alex advice was correct. If it wasn't logo, you should find what image is missing.

I tested it and in my case, it redirects correctly to https version.   You can try it on e107.org too, just thumb.php?src=&aw=144&ah=144  and you will see.

**Reply by rwjework** [reply]
Thanks Jimmi, I will update from Version 2.3.0 to 2.3.3 and see if that fixes the issue.

**Reply by rwjework** [reply]
I have updated to 2.3.2 and the security has sorted itself out. Thanks for your help
